src/applications/appeals/shared/components/DateSubmitted.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FORMAT_READABLE } from '../constants';
+
+export const DateSubmitted = props => (
```

Comment:
small nit, not blocking — if you destructured the props here, you can do `submitDate.format(...)` below

```suggestion
export const DateSubmitted = ({ submitDate }) => (
```